docs/source/snippets/data/transforms/datamodule_train_eval_cli.yaml

```diff
@@ -0,0 +1,43 @@
+class_path: anomalib.data.MVTec
+init_args:
+  root: ./datasets/MVTec
+  category: bottle
+  image_size: [256, 256]
+  train_batch_size: 32
+  eval_batch_size: 32
+  num_workers: 8
+  task: segmentation
+  test_split_mode: from_dir
+  test_split_ratio: 0.2
+  val_split_mode: same_as_test
+  val_split_ratio: 0.5
+  seed: null
+  eval_transform:
+    class_path: torchvision.transforms.v2.Compose
+    init_args:
+      transforms:
+        - class_path: torchvision.transforms.v2.RandomAdjustSharpness
+          init_args:
+            sharpness_factor: 0.7
+            p: 0.5
+        - class_path: torchvision.transforms.v2.RandomHorizontalFlip
+          init_args:
+            p: 0.5
+        - class_path: torchvision.transforms.v2.Resize
+          init_args:
+            size: [256, 256]
+        - class_path: torchvision.transforms.v2.Normalize
+          init_args:
+            mean: [0.485, 0.456, 0.406]
+            std: [0.229, 0.224, 0.225]
```

Comment:
I was not aware if we support this. Are we certain that this is supported? @djdameln @ashwinvaidya17 ?

Reply:
I tested this and it works :)